### PR TITLE
Dropdown max_player fix

### PR DESCRIPTION
**Pull request by KingKnecht**
Get index of selected value from list of max_player.

### CODE REVIEW

**Comment by KingKnecht** [comment]
guess I should have rebased or smt. Only the last commit is relevant to the problem. All other commits are just the merge of the new master into my branch. I don't get Git 🤣. Sorry for the mess...

**Comment by dickreuter** [comment]
with those changes it doesn't save alterations of that dropdown anymore, no?

**Comment by KingKnecht** [comment]
It does. Previously it did it twice. That was another "bug". I will send you a video on discord of how it works now before you merge.

File: poker/scraper/table_setup_actions_and_signals.py [hunk 1]
```diff
@@ -261,11 +260,6 @@ def _connect_cards_with_save_slot(self):
         button_show_property = getattr(self.ui, 'covered_card_show')
         button_show_property.clicked.connect(lambda state: self.load_image('covered_card'))
 
-    def _connect_save_max_players_with_save_slot(self):
```
Comment (dickreuter) [inline poker/scraper/table_setup_actions_and_signals.py:264]: if this is removed, how does it save any changes?

Comment (KingKnecht) [reply poker/scraper/table_setup_actions_and_signals.py:264]: I don't know how to cite a code line here. But in Line 101:
`self.ui.max_players.currentIndexChanged[int].connect(lambda: self._save_max_players())`
there is already a connect for a changed index. The value has been saved twice, when selected Index changed.

**Comment by KingKnecht** [comment]
@dickreuter This is what I came up with. I don't know. Git hates me. If this is all bullshit I will start from scratch. Creating a new fork and fix the bug again. I have a copy of my changes.

**Pull request merged**